sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -3576,7 +3612,7 @@ class DataSourceV2SQLSuiteV1Filter
           s"REPLACE TABLE tab (col1 DOUBLE DEFAULT rand()) USING $v2Source")
         assert(exception.getSqlState == "42623")
         assert(exception.errorClass.get == "INVALID_DEFAULT_VALUE.NON_DETERMINISTIC")
-        assert(exception.messageParameters("statement") == "CREATE TABLE")
+        assert(exception.messageParameters("statement") == "REPLACE TABLE")
```

Comment:
This looks like an error message bug fix, right?

Reply:
yea , my mistake before :)